ganga/GangaCore/GPIDev/Lib/Job/Job.py

```diff
@@ -1639,9 +1639,22 @@ def removeFiles(this_file):
                 if getName(this_file) in getConfig('Output')['AutoRemoveFileTypes'] and hasattr(this_file, '_auto_remove'):
                     this_file._auto_remove()
 
+            _filesToRemove = []
             for sj in self.subjobs:
-                map(removeFiles, sj.outputfiles)
-            map(removeFiles, self.outputfiles)
+                for _f in sj.outputfiles:
+                    if _f.containsWildcards() and hasattr(_f, 'subfiles') and _f.subfiles:
```

Comment:
Would look nicer if the lines that are here and repeated below was in a little helper function.